app/components/cookies_banner_component.rb

```diff
@@ -0,0 +1,21 @@
+class CookiesBannerComponent < GovukComponent::Base
+  include ViewComponent::SlotableV2
+  attr_reader :create_path, :reject_path, :preferences_path
+
+  renders_one :body
+  renders_one :actions
+
+  def initialize(create_path:, reject_path:, preferences_path:, classes: [], html_attributes: {})
+    super(classes: classes, html_attributes: html_attributes)
+
+    @create_path = create_path
+    @reject_path = reject_path
+    @preferences_path = preferences_path
+  end
+
+  private
+
+  def default_classes
```

Comment:
Where is default_classes used?

Reply:
It is something that extending from gov component gives and just classes that are applied to component container